### PR TITLE
add mastodon icon to settings/user

### PR DESCRIPTION
**Pull request by violoncelloCH**
as discussed in #6714 
please review @nickvergessen @jospoortvliet @jancborchardt

### CODE REVIEW

File: settings/templates/settings/personal/development.notice.php [hunk 1]
```diff
@@ -23,42 +23,49 @@
...
 				'<a target="_blank" rel="noreferrer noopener" href="https://www.facebook.com/Nextclouders/">',
 				'<a target="_blank" rel="noreferrer noopener" href="https://twitter.com/nextclouders">',
+				'<a target="_blank" rel="noreferrer noopener" href="https://mastodon.xyz/@nextcloud">',
 				'<a target="_blank" rel="noreferrer noopener" href="https://nextcloud.com/news/">',
 				'<a target="_blank" rel="noreferrer noopener" href="https://newsletter.nextcloud.com/?p=subscribe&amp;id=1">',
 				'</a>',
 				$l->t('Follow us on Google+'),
 				$l->t('Like our Facebook page'),
 				$l->t('Follow us on Twitter'),
+				$l->t('(Remote-)Follow us on Mastodon'),
```
Comment (schiessle) [inline settings/templates/settings/personal/development.notice.php]: Why `(Remote-)Follow`?

Comment (violoncelloch) [reply settings/templates/settings/personal/development.notice.php]: I thought to emphasize federation... at least this is how mastodon calls it if you open a profile page and want to follow
But I can change this back to simply "Follow"

Comment (schiessle) [reply settings/templates/settings/personal/development.notice.php]: Yes I know. I'm using Mastodon as well :slightly_smiling_face: But I think here we don't need to make the distinction. It is to technical. Thanks for changing it back! :+1:

**Review by blizzz — APPROVED** [review]

**Review by nickvergessen — APPROVED** [review]

**Comment by jancborchardt** [comment]
@schiessle @blizzz @nickvergessen please add @violoncelloch to the Nextcloud organization to welcome them! :tada: :slightly_smiling_face:

**Comment by violoncelloch** [comment]
Ohhh, thank you @jancborchardt I'm very happy about this 😊🥂🎉

**Comment by nickvergessen** [comment]
Done, but also note @jancborchardt you are an owner too.
Just go to https://github.com/orgs/nextcloud/people and click `Invite member`

**Pull request merged**